stripe/resource.py

```diff
@@ -17,7 +17,7 @@ def convert_to_stripe_object(resp, api_key, account):
              'fee_refund': ApplicationFeeRefund,
              'bitcoin_receiver': BitcoinReceiver,
              'bitcoin_transaction': BitcoinTransaction,
-             'transfer_reversal': Reversal}
+             'transfer_reversal': Reversal, 'dispute': Dispute}
```

Comment:
You asked me to turn on nit-picky mode... I wish this list and all the classes in this file were sorted in the same order (preferably alphabetical). That's my fault for not doing it in the first place though, not worth polluting this diff.